src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -59,11 +120,18 @@ export default function Banner() {
           <Typography variant="h1" className={classes.h1}>
             {'Telescope'}
           </Typography>
+
+          <Typography variant="caption" className={classes.stats}>
+            <RetrieveStats />
+          </Typography>
         </ThemeProvider>
-        <div className="version">v {Version.version}</div>
+        <div className="version" className={classes.version}>
+          v {Version.version}
+        </div>
+
         <div className="icon">
           <ScrollDown>
-            <Fab color="primary" size="medium" aria-label="scroll-down">
```

Comment:
removing `size="medium` makes the blue circle the same size but not the arrow itself, is there an easy way to do it or should it be fixed in #830 ?

Reply:
Corrected arrow in latest, completely forgot about the arrow :laughing:

Reply:
Thanks for the quick fix